<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="SvgView" name="view" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>546</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="AbortWidget" name="abort" native="true">
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionFileOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionLayoutDot">
   <property name="text">
    <string>Dot</string>
   </property>
  </action>
  <action name="actionTwopi">
   <property name="text">
    <string>TwoPi</string>
   </property>
  </action>
  <action name="actionLayoutTwopi">
   <property name="text">
    <string>Twopi</string>
   </property>
  </action>
  <action name="actionLayoutCirco">
   <property name="text">
    <string>Circo</string>
   </property>
  </action>
  <action name="actionLayoutNeato">
   <property name="text">
    <string>Neato</string>
   </property>
  </action>
  <action name="actionLayoutFdp">
   <property name="text">
    <string>Fdp</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AbortWidget</class>
   <extends>QWidget</extends>
   <header>abortwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SvgView</class>
   <extends>QWidget</extends>
   <header>svgview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
